app/javascript/article-form/components/ArticleCoverImage.jsx

```diff
@@ -169,6 +163,9 @@ export class ArticleCoverImage extends Component {
         }
       : {};
 
+    // Native Bridge messages come through ForemMobile events
+    document.addEventListener('ForemMobile', this.handleNativeMessage);
```

Comment:
Instead of the hidden `input` field we are moving towards [CustomEvents](https://developer.mozilla.org/en-US/docs/Web/Events/Creating_and_triggering_events)